examples/jsm/utils/GPUStatsPanel.js

```diff
@@ -86,7 +86,7 @@ export class GPUStatsPanel extends Stats.Panel {
 					this.activeQueries --;
 
 
-				} else {
+				} else if ( !gl.isContextLost() ) {
```

Comment:
nit: can we add a space after the `!` to align with the project code style? Otherwise looks fine to me

Reply:
Fixed!